rpc/tests/test_seth_rpc.py

```diff
@@ -238,6 +239,9 @@ def test_get_block_by_hash(self):
         self.assertEqual(result["stateRoot"], "0x" + self.state_root)
         self.assertEqual(result["gasUsed"], hex(self.gas))
         self.assertEqual(result["transactions"][0], "0x" + self.txn_id)
+        self.assertEqual(result["status"], "0x" + self.status)
```

Comment:
Looks like this line is causing issues in Jenkins atm:

https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-seth/view/change-requests/job/PR-77/7/console